### DISCUSSION TITLE
What does "Close the download bar and allow an alternative API to download files" do?

### DISCUSSION THREAD

**Discussion by idkhowisyourday** [gdh1995/vimium-c#995] [Q&A]
![image](https://github.com/gdh1995/vimium-c/assets/90016459/d62050e4-e3df-4ffa-b40d-a14cd9f7a494)
What does this option do? I'm using microsoft edge so the download bar isn't there anymore, so is this deprecated?

**Comment by gdh1995** [comment]
It's still necessary for some commands of Vimium C - `LinkHints.activateDownloadLink` and `LinkHints.activateDownloadImage` will require this permission to trigger downloading when in some situations and/or key mapping parameters.

**Comment by gdh1995** [comment]
I'll update the description on MS Edge.
